sstables/sstables_manager.cc

```diff
@@ -61,4 +67,44 @@ sstable_writer_config sstables_manager::configure_writer() const {
     return cfg;
 }
 
+void sstables_manager::add(sstable* sst) {
+    _active.push_back(*sst);
+    smlogger.debug("activated sstable, {} active sstables", value_of([this] { return _active.size(); }));
+}
+
+void sstables_manager::inactivate(sstable* sst) {
+    // At this point, sst has a reference count of zero, since we got here from
+    // lw_shared_ptr_deleter<sstables::sstable>::dispose().
+    smlogger.debug("deactivating sstable, {} active sstables", value_of([this] { return _active.size(); }));
+    _active.erase(_active.iterator_to(*sst));
+    _undergoing_close.push_back(*sst);
+    // guard against sstable::close_files() calling shared_from_this() and immediately destroying
+    // the result, which will dispose of the sstable recursively
+    auto ptr = sst->shared_from_this();
+    (void)sst->close_files().finally([ptr] {
+        smlogger.debug("deletion complete");
```

Comment:
let's add more context to "deletion complete", sstable's path would be enough to disambiguate

Reply:
Sure.